RELEASE.md

```diff
@@ -293,9 +293,9 @@ Wait for a few minutes so that supervisors on all the workers can update to the
 1. Update `RUSTFMT_VERSION` in the root of the habitat repo.
 1. Locally install the nightly toolchain and update the formatting. For example:
     ```
-    rustup toolchain install $(cat RUSTFMT_VERSION)
+    rustup toolchain install $(< RUSTFMT_VERSION)
     rustup component add --toolchain $(cat RUSTFMT_VERSION) rustfmt
-    cargo +$(RUSTFMT_VERSION) fmt
+    cargo +$(< RUSTFMT_VERSION) fmt
```

Comment:
Nitpick, but this syntax isn't portable across shells.  In particular, it doesn't work on `fish`.  I suppose this particular nitpick is slightly negated in the sense that the `$()` subshell syntax isn't supported on fish either, and maybe we're not interested in portable shell code in our instructions.  But with `cat`, removing the `$` turns it into a usable command, vs with `<`, it's unusable unless you replace that too.  So this way requires an extra thought process and substitution for those of us that don't use `bash`.  All of which is to say, whatever.  I'm not super invested in this one way or the other, and honestly, fish users are probably pretty used to converting bashisms to fishisms.  But I thought it was worth pointing out.

Reply:
Fair point. I also didn't notice the `cat` on the line above; consistency would be good.

I'll tweak this.